x/ethbridge/client/cli/tx.go

```diff
@@ -100,21 +120,31 @@ func GetCmdBurn(cdc *codec.Codec) *cobra.Command {
 				return err
 			}
 
-			tokenContractString := viper.GetString(types.FlagTokenContractAddr)
-			tokenContract := types.NewEthereumAddress(tokenContractString)
-
 			cosmosSender, err := sdk.AccAddressFromBech32(args[0])
 			if err != nil {
 				return err
 			}
 
+			if !common.IsHexAddress(args[1]) {
+				return errors.Errorf("invalid [ethereum-receiver-address]: %s", args[1])
+			}
 			ethereumReceiver := types.NewEthereumAddress(args[1])
-			amount, err := sdk.ParseCoins(args[2])
+
+			var digitCheck = regexp.MustCompile(`^[0-9]+$`)
+			if !digitCheck.MatchString(args[2]) {
+				return types.ErrInvalidAmount
+			}
+			amount, err := strconv.ParseInt(args[2], 10, 64)
 			if err != nil {
 				return err
 			}
+			if amount <= 0 {
+				return types.ErrInvalidAmount
+			}
+
+			symbol := args[3]
```

Comment:
do we need to check if this string is empty?

Reply:
This is effectively checked by the prefix/suffix checks in MsgBurn ValidateBasic, but yeh may b good to check here in CLI too.